public/data/feedList.json

```
@@ -0,0 +1,122 @@
+[
+  {
+    "id": 1,
+    "name": "rocket.racoon",
+    "profile": "../../../images/minjeong/circe-denyer-W_8Tivmufmo-unsplash.png",
+    "feedImg": "../../../images/minjeong/racoon.png",
+    "likeImg": "../../../images/minjeong/tree.png",
```

```suggestion
    "profile": "./images/minjeong/circe-denyer-W_8Tivmufmo-unsplash.png",
    "feedImg": "/images/minjeong/racoon.png",
    "likeImg": "/images/minjeong/tree.png",
```
이미지 경로 수정하셔도 될 것 같습니다 .